src/Backoffice/Airlines/App/Controllers/StoreAirlineController.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Backoffice\Airlines\App\Controllers;
+
+use Illuminate\Http\JsonResponse;
+use Lightit\Backoffice\Airlines\App\Request\StoreAirlineRequest;
+use Lightit\Backoffice\Airlines\App\Transformers\AirlineTransformer;
+use Lightit\Backoffice\Airlines\Domain\Actions\StoreAirlineAction;
+
+class StoreAirlineController
+{
+    public function __invoke(
+        StoreAirlineRequest $request,
+        StoreAirlineAction $storeAirlineAction,
+    ): JsonResponse {
+        $airline = $storeAirlineAction->execute($request->toDto());
+
+        return responder()
+            ->success($airline, AirlineTransformer::class)
+            ->respond();
```

Comment:
```suggestion
            ->respond(JsonResponse::HTTP_CREATED);
```
For endpoints that store data, they should respond with the 201 HTTP status code.